lib/include/public/LogManagerBase.hpp

```diff
@@ -716,10 +716,11 @@ namespace MAT_NS_BEGIN
 
         static void WaitPause()
         {
-            auto instance = GetInstance();
-            if (instance) {
+            auto managerInstance = GetInstance();
+            if (managerInstance)
+            {
```

Comment:
Looks like the clang-tidy rules changed this. LMK if you want that reverted.

Reply:
I think this should be fine.